wdtk-client/src/main/java/org/wikidata/wdtk/client/DumpProcessingAction.java

```diff
@@ -29,16 +29,16 @@
  * {@link EntityDocumentDumpProcessor}. Additional methods provide a generic
  * interface for setting options and other auxiliary information that the action
  * might require.
- *
+ * 
  * @author Markus Kroetzsch
- *
+ * 
  */
 public interface DumpProcessingAction extends EntityDocumentDumpProcessor {
 
 	/**
 	 * Returns true if processing requires a {@link Sites} object to be set.
 	 * This is the case for some operations that process site links.
-	 *
+	 * 
```

Comment:
Any idea why these changes are happening? I am also seeing this on my machine and it creates a lot of pointless changes. Is Maven doing this or Eclipse?